docs/technical_reference/unit_models/coag_floc_model.rst

```diff
@@ -0,0 +1,160 @@
+Coagulation-Flocculation
+========================
+
+Introduction
+------------
+
+Coagulation-Flocculation is a water treatment process  designed to remove total suspended
+solids (TSS, primarily as natural organic matter) from solution by converting those small,

```

Comment:
I would remove "primarily as natural organic matter", as it can contain inorganic colloids depending on the specific water.  Technically a part of "dissolved organic matter" may also be removed via forming particles with coagulants.  Thus, consider changing "converting those small, suspended .." to "converting  small suspended and colloidal particulate matter and some dissolved natural organic matter to ..."